core/src/09-special-equipment/01-chapter-nine.md

```diff
@@ -840,3 +842,81 @@ ___
 | Giant Scorpion | 4 | 35' | Independent 1 | Agility 5 <br>Perception 5 | Bane Focus I (Persistent Damage) | 20 | 2 |  Guard: 15 <br>Toughness: 15 <br>Resolve: 18 |
 
 </div>
+
+
+
+
+
+
+
+
+
+
+Cody JA Brown
+Dan Scantland
+Daniel Marjoram
+Deven Morgan
+Donna Waldrop
+Elvenoob
+Giuliano Michelon
+Gregg Miller
+John Spearing
+Justin Packwood
+Kerry Wilcox
+KICK in a BOX Inc.
+Latt
+Lone Shark Games
+Marcelino
+Mario
```

Comment:
It is indeed a different Mario 😄  

I've done people who didn't fill out the survey the courtesy of including them anyway, and our friend Mario put only that down for his contact info ;)